transformer/cloudfoundrytransformer.go

```diff
@@ -263,21 +299,43 @@ func (t *CloudFoundry) prioritizeAndAddEnvironmentVariables(cfApp collecttypes.C
 			flattenedEnvList := flattenVcapServiceVariables(valueStr)
 			for _, env := range flattenedEnvList {
 				envOrderMap[env.Name] = env
+				vcapEnvMap[env.Name] = []byte(env.Value)
 			}
 		}
+		dbEnv, err := createDbEnvironmentVariable(valueStr, serviceName)
+		if err != nil {
+			logrus.Errorf("Could not create DATABASE_URL environment: %v", err)
+		} else {
+			envOrderMap[dbEnv.Name] = dbEnv
+			vcapEnvMap[dbEnv.Name] = []byte(dbEnv.Value)
+		}
 		envOrderMap[varname] = core.EnvVar{Name: varname, Value: valueStr}
+		vcapEnvMap[varname] = []byte(valueStr)
```

Comment:
why would this need to be in vcapenvmap?

Reply:
Because vcapenvmap contains all variables which go into the secret. This includes the vcap flattened environment variables as well as the ones like DATABASE_URL generated using vcap values.

Reply:
done based on clarification.